src/js/controller/Setup.js

```diff
@@ -20,11 +20,15 @@ define([
 ], function(utils, Playlist, Skin, eventdispatcher, _, events) {
 
     var Setup = function(model, view) {
-        var _model = model,
+        var _this = this,
```

Comment:
Instead of using _this and _cancelled, you could bind 'this' into the setupTimeoutHandler on line 69